handlers/dd/dataset/handler_test.go

```diff
@@ -70,7 +70,7 @@ func TestHandler(t *testing.T) {
 
 	Convey("Dataset title is rendered", t, func() {
 		recorder := httptest.NewRecorder()
-		rdr := bytes.NewReader([]byte(`{"dataset":{"id":"ID1","title":"A Test Dataset"}}`))
+		rdr := bytes.NewReader([]byte(`{"dataset":{"id":"ID1","title":"A Test Dataset","dataDiscoveryAssetsPath":"foobar"}}`))
```

Comment:
Fair point, it was leftover from previous attempts to fix the broken test, so can be removed.